sql-to-dbsp-compiler/lib/sqllib/src/geopoint.rs

```diff
@@ -72,3 +73,9 @@ pub fn make_geopointN_dN_dN(left: Option<F64>, right: Option<F64>) -> Option<Geo
         (Some(x), Some(y)) => make_geopointN_d_d(x, y),
     }
 }
+
+pub fn st_distance_geopoint_geopoint(left: GeoPoint, right: GeoPoint) -> F64 {
```

Comment:
Why do we need to pass these arguments by value?

Reply:
Three more functions are generated by a macro, with signatures like
`st_distance_geopointN_geopoint(left: Option<GeoPoint>, right: GeoPoint) -> Option<F64>`. The signature of the other functions is related to this one. They all also call this function.